src/contracts/contract.ts

```diff
@@ -51,18 +51,18 @@ export class Contract {
 
   public paramsBuilder({
     method,
+    blockHeight,
     params
   }: {
     method: string;
+    blockHeight?: number;
     params?: {
       [key: string]: any;
     };
   }) {
-    return new IconBuilder.CallBuilder()
-      .to(this.address)
-      .method(method)
-      .params(params)
-      .build();
+    const rawTx = new IconBuilder.CallBuilder().to(this.address).method(method).params(params).build();
```

Comment:
IconBuilder instance doesn't have a function to set height?

Reply:
I think it doesn't, haven't found anything in the code nor the docs